jaxopt/_src/perturbations.py

```diff
@@ -57,6 +58,11 @@ def make_perturbed_argmax(argmax_fun: Callable[[jax.Array], jax.Array],
     noise: a distribution object that must implement a sample function and a
       log-pdf of the desired distribution, similar to the examples above.
       Default is Gumbel distribution.
+    control_variate : Subtract a control variate (the unperturbed argmax) from
```

Comment:
I would change this a bit, starting with 'a boolean, indicates whether, ...' to specify the nature - `control_variate` is not a function that substracts.